### DISCUSSION TITLE
The Command Palette does not format backticks in PR names

### DISCUSSION THREAD

**Discussion by yoav-lavi** [community/community#6902] [Other Feature Feedback, Questions, & Ideas] [answered]
The new Command Palette does not format backticks (``) in PR names and leaves them as is, which is at odds with the rest of the GitHub UI

**Comment by gaknoll** [comment, marked as answer]
Appreciate all the great feedback and bug reports @yoav-lavi 🙏🏽 
Great catch on this one, I've added this to our backlog to investigate

**Reply by yoav-lavi** [reply]
Thanks @gaknoll!
Are you planning to add a command palette topic to this repo?

**Reply by gaknoll** [reply]
@yoav-lavi We aren't planning a Command Palette topic in github/feedback at this time, but we do have a category over in github/community that we are also using for collecting feedback: https://github.community/c/command-palette/71

**Reply by yoav-lavi** [reply]
@gaknoll Yeah I noticed that, the reason I ended up sending the feedback here is that I preferred not to approve the "Act on your behalf" permission for the github.community auth (I'm aware that it's also a GitHub product and that it's probably harmless, just felt a bit weird that user auth would need that permission)

A bit curious as to why there's a different platform for feedback on this feature, is it because it's in beta?

Anyhow thanks for the prompt responses! Will let you know if I run into anything else, really like the idea of a command palette for GitHub!

**Reply by gaknoll** [reply]
@yoav-lavi we're still using both depending on the product! So whichever is easiest for you, I'm monitoring both
